### PR TITLE
Add support link to community page, and add Matrix to support page

### PR DESCRIPTION
**Pull request by GregSutcliffe**
The community page needs a bigger overhaul IMO, but for now this is a small update that makes the IRC/lists a little more discoverable. Also adds the Matrix details.

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ There are also a few extra medium-specific guidelines, listed with each type of
 ### Reporting Violations
 
 Should you need to speak with someone about another Foreman member's behaviour,
-email the Community Lead, *Greg Sutcliffe* (greg.sutcliffe at gmail.com). You
-may also CC or email *Ohad Levy* (ohad.levy at gmail.com).  We'll do our best
-to listen to your issue fairly and take appropriate action.
+email the Community Lead, *Greg Sutcliffe* (greg at emeraldreverie.org, male)
+and/or *Ori Rabin* (orabin at redhat.com, female).  We'll do our best to listen
```
Comment (GregSutcliffe) [inline support.md]: @orrabin are you OK with gender in this? From a diversity point of view I think it's important that people reporting violations are made comfortable in doing so, and I've heard this information can help.

File: support.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ There are also a few extra medium-specific guidelines, listed with each type of
 ### Reporting Violations
 
 Should you need to speak with someone about another Foreman member's behaviour,
-email the Community Lead, *Greg Sutcliffe* (greg.sutcliffe at gmail.com). You
-may also CC or email *Ohad Levy* (ohad.levy at gmail.com).  We'll do our best
-to listen to your issue fairly and take appropriate action.
+email the Community Lead, *Greg Sutcliffe* (greg at emeraldreverie.org, male)
```
Comment (ekohl) [inline support.md]: It may be a bit odd to have a title for you but not for Ori.

Comment (GregSutcliffe) [reply support.md]: True, I'll delete mine, it's not relevant anyway

Comment (GregSutcliffe) [reply support.md]: Done in the Github UI as im on another branch right now, so its an extra commit. squash on merge pls :P

**Review by ekohl — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Ugh, forgot to merge this, thanks for the review @ekohl. @orrabin has confirmed that text is OK, so merging this.

**Pull request merged**